src/pages/components/form/usage.mdx

```diff
@@ -407,6 +423,34 @@ pattern.
 | [Designing for longer forms](/patterns/forms-pattern/#designing-for-longer-forms) | Techniques to help make longer forms less overwhelming, including guidance for accordion and multistep forms.   |
 | [Form variants](/patterns/forms-pattern/#variants)                                | Forms may be presented as dedicated pages, side panels, or dialogs depending on the use case and the situation. |
 
+## AI presence
+
+Form has a modification that takes on the AI visual styling when AI is present
+in the form. The AI variants of a form function the same as the normal variants
+except for the addition of the AI slug, which is both a visual indicator and the
+trigger for the explainability popover.
+
+The AI presence in form can appear in two ways: broadly, over the whole form, or
+focused, just in certain parts of the form. When the entire form is styled for
+AI, it indicates to the user that AI is present in all aspects of the form. If
+only a limited amount of the form content is generated by AI, then only those
+components should receive the AI presence styling, not the entire form.
+
+More information about designing for AI guidelines is coming soon.
+
+<Row>
+<Column colLg={12}>
+
+![Example of form with full and compoment based AI presence](images/usage-ai-presence-form.png)
```

Comment:
Ok this image is actually huge at 12 columns and it becomes hard to see and the spacing is very tight. I think what we need to do here is scale the forms down (by ~75%) which we normally don't do but I think it needs to here. And maybe removing the text area will help with height. I added an [example](https://www.figma.com/design/9J3QEJ717fLUa7tzlWVMMu/Usage-and-Style-images?node-id=2036-31868&t=tcnwHu1HbcbrAkUV-1) in the file.